pkgs/tools/security/nmap/default.nix

```diff
@@ -27,7 +27,14 @@ in stdenv.mkDerivation rec {
     sha256 = "063fg8adx23l4irrh5kn57hsmi1xvjkar4vm4k6g94ppan4hcyw4";
   };
 
-  patches = ./zenmap.patch;
+  patches = [ ./zenmap.patch ]
+    ++ optional stdenv.cc.isClang (
```

Comment:
very minorly, [use `optionals` here](https://discourse.nixos.org/t/type-safety-in-nix/2871)

Reply:
The thread isn't clear: is `optionals [(fetchpatch {})]` preferred over `optional (fetchpatch {})` due to the increased type safety?

Reply:
thanks for humor me, this is more of a personal preference, since it's easy to mix optional/optionals (e.g. https://github.com/NixOS/nixpkgs/pull/60123) I'm inclined to use `optionals` even when is not necessary